src/openrct2/ride/ride_ratings.c

```diff
@@ -904,10 +904,10 @@ static uint32 ride_ratings_get_proximity_score()
 }
 
 /**
- * Seems to calculate how much of the track is sheltered in eighths.
+ * Calculates how much of the track is sheltered in eighths.
```

Comment:
I think this function needs refactoring for its outputs. The `sint32` is comprised of two `uint8`'s one of which is the sheltered eights and the other is the sheltered eights or 7 depending on if the ride is a covered ride.